utils/utils.go

```diff
@@ -29,3 +30,29 @@ func CompileAndRunRegex(regex string, base string, t *testing.T, err string, sho
 		t.Errorf(err)
 	}
 }
+
+// adapted from https://gist.github.com/albrow/5882501
+func UserConfirmation(message string) bool {
+	fmt.Println(message)
+
+	var input string
+	_, err := fmt.Scanln(&input)
+	if err != nil {
+		// should maybe log something here
+		return false
+	}
+	yesResponses := []string{"y", "Y", "yes", "Yes", "YES"}
```

Comment:
nit: move these out of the function to a package level var?

Reply:
done